CHANGELOG.md

```diff
@@ -7,6 +7,195 @@ documentation before upgrading to a new release.
+| mysql                            | --extra-opts               | mysql\_extra\_opts                       | #9197 |
+| nrpe                             | -3                         | nrpe\_version\_3                         | #9296 |
+| nrpe                             | -D                         | nrpe\_no\_logging                        | #9016 |
+| nrpe                             | -P                         | nrpe\_payload\_size                      | #9032 |
+| pgsql                            | --extra-opts               | pgsql\_extra\_opts                       | #9197 |
+| postgres                         | $PGCONTROLDATA (env. var.) | postgres\_pgcontroldata                  | #8929 |
+| postgres                         | --datadir                  | postgres\_datadir                        | #8924 |
+| postgres                         | --language                 | postgres\_language                       | #8924 |
+| postgres                         | --perflimit                | postgres\_perflimit                      | #8924 |
+| ssl\_cert                        | --ignore-host-cn           | ssl\_cert\_ignore\_host\_cn              | #9512 |
+| ssl\_cert                        | --ignore-ocsp-errors       | ssl\_cert\_ignore\_ocsp\_errors          | #9512 |
+| ssl\_cert                        | --ignore-ocsp-timeout      | ssl\_cert\_ignore\_ocsp\_timeout         | #9512 |
+| ssl\_cert                        | --ignore-tls-renegotiation | ssl\_cert\_ignore\_tls\_renegotiation    | #9042 |
+| ssl\_cert                        | --proxy                    | ssl\_cert\_proxy                         | #8927 |
+| tcp                              | --sni                      | tcp\_sni                                 | #9347 |
+| vmware-esx-dc-runtime-tools      | --no\_vm\_tools\_ok        | vmware\_novmtools                        | #9611 |
+| vmware-esx-soap-vm-runtime-tools | --no\_vm\_tools\_ok        | vmware\_novmtools                        | #9611 |
+
+### Miscellaneous
+
+* Require GCC 7+ for building to enable C++17. #9133 #9485 #9489
+* Require CMake v2.8.12+ for building.
+  (Compatibility with older ones will be removed from a future CMake version.) #9706
+* Repair config reload on OpenBSD by using waitpid(2), not a SIGCHLD handler. #9518
+* Ignore SIGHUP in main process to allow `/etc/rc.d/icinga2 reload`
+  sending it to all Icinga 2 processes on OpenBSD. #9622
+* Fix crash in debug build on macOS when API and debug log are enabled. #9497
+* Update Boost shipped on Windows to v1.82. #9761
+* Update OpenSSL shipped on Windows to v3.0.9. #9787
+* Update vendored https://github.com/nlohmann/json to v3.9.1. #9675
+* Update vendored https://github.com/nemtrif/utfcpp to v3.2.3. #9683
+* Documentation: several fixes and improvements.  #8954 #9741 #9763 #9767 #9769 #9777
+* Several code quality improvements. #8815 #9106 #9250
+  #9508 #9517 #9537 #9594 #9605 #9606 #9641 #9658 #9702 #9717 #9738
```

Comment:
At first glance, #9702 looks like it would have a similar effect like disabling the perfdata writers on Windows. Can you verify whether this removes other components on Windows and if it does, it probably makes sense to include them in the list above as well.

Reply:
No, neither at first glance, not at a closer look. That PR does exactly what its description says, not more, not less.

Reply:
PR description:

> CMakeLists.txt: provide sane defaults for which subsystems to build not to have to explicitly disable particular ones in multiple locations.

Makes no statement which of them were already disables before. But taking a closer look at the diff of that PR, it only disables those on Windows in `CMakeLists.txt` that were previously disabled in `configure.ps1`.